db/migrate/20240514072741_add_method_to_recurring_transaction_rules.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddMethodToRecurringTransactionRules < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :recurring_transaction_rules, :rule_type, :trigger
+    add_column :recurring_transaction_rules, :method, :integer, null: false, default: 0
```

Comment:
I don't see where method is used in the PR. Is it a leftover of a previous implementation that didn't make it?

Reply:
It was in the description, but yep this new column `action` has been added as well to not create another migration in the next PR

Reply:
Oh yeah I missed it. The new column seem to be named `method` tho, not `action`.

Reply:
Oh my bad 🤦 😅